roles/redpanda_broker/defaults/main.yml

```diff
@@ -35,13 +35,18 @@ airgap_copy_dest: "/tmp"
 # the rpms
 rp_key_rpm: "{{ redpanda_base_url }}/public/redpanda/gpg.988A7B0A4918BC85.key"
 rp_key_rpm_unstable: "{{ redpanda_base_url }}/E4xN1tVe3Xy60GTx/redpanda-unstable/gpg.39C20393EC2E8747.key"
-rp_standard_rpm: "{{ redpanda_base_url }}/public/redpanda/rpm/{{ (ansible_distribution == 'RedHat') | ternary('el', ansible_distribution | lower) }}/{{ ansible_distribution_major_version }}/$basearch"
-rp_standard_rpm_unstable: "{{ redpanda_base_url }}/E4xN1tVe3Xy60GTx/redpanda-unstable/rpm/{{ (ansible_distribution == 'RedHat') | ternary('el', ansible_distribution | lower) }}/{{ ansible_distribution_major_version }}/$basearch"
-rp_noarch_rpm: "{{ redpanda_base_url }}/public/redpanda/rpm/{{ (ansible_distribution == 'RedHat') | ternary('el', ansible_distribution | lower) }}/{{ ansible_distribution_major_version }}/noarch"
-rp_noarch_rpm_unstable: "{{ redpanda_base_url }}/E4xN1tVe3Xy60GTx/redpanda-unstable/rpm/{{ (ansible_distribution == 'RedHat') | ternary('el', ansible_distribution | lower) }}/{{ ansible_distribution_major_version }}/noarch"
-rp_source_rpm: "{{ redpanda_base_url }}/public/redpanda/rpm/{{ (ansible_distribution == 'RedHat') | ternary('el', ansible_distribution | lower) }}/{{ ansible_distribution_major_version }}/SRPMS"
-rp_source_rpm_unstable: "{{ redpanda_base_url }}/E4xN1tVe3Xy60GTx/redpanda-unstable/rpm/{{ (ansible_distribution == 'RedHat') | ternary('el', ansible_distribution | lower) }}/{{ ansible_distribution_major_version }}/SRPMS"
+rp_standard_rpm: "{{ redpanda_base_url }}/public/redpanda/rpm/{{ rp_rpm_distro_map[ansible_distribution] | default('el') }}/{{ ansible_distribution_major_version }}/$basearch"
+rp_standard_rpm_unstable: "{{ redpanda_base_url }}/E4xN1tVe3Xy60GTx/redpanda-unstable/rpm/el/{{ ansible_distribution_major_version }}/$basearch"
```

Comment:
Just to be sure, this works with unstable on non rhel OS even tho it is forced to el? I'll check tomorrow to be sure, but if you've already tested it it'd be great to know!

Reply:
I didn't, it's a miss, I've fixed it now.